index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300&display=swap" rel="stylesheet"> 
+    <script src="https://kit.fontawesome.com/7bc3c221d9.js" crossorigin="anonymous"></script>
+    <title>Practice</title>
+</head>
+<body>
+                    <!--NAVBAR-->
+<div class="navbar">
+    <div class="initial">
+    <img src="images/line.png" alt="line" >
+    <img src="images/YouTube.png" alt="line">
+</div>
+<div class="search">
+        <input type="text"  placeholder="Search.." > 
+    <div class="searchbtn">
```

Comment:
Here should be a `<button>` tag. 
![image](https://user-images.githubusercontent.com/56762529/90577418-3d1f6380-e197-11ea-9905-05a4fd4e4de0.png)